xc7/primitives/bram/bram.pb_type.xml

```diff
@@ -311,7 +311,12 @@ reason, BRAM tiles use a "fused sites" mode to accomidate the cross site logic.
      <pb_type name="RAMB18E1_Y0_IN" num_pb="1" blif_model=".subckt RAMB18E1_VPR">
       <xi:include href="ramb18e1.pb_type.xml" xpointer="xpointer(pb_type/child::node()[local-name()!='metadata'])" />
       <metadata>
-       <xi:include href="ramb18e1.pb_type.xml" xpointer="xpointer(pb_type/metadata/child::node())" />
+       <xi:include href="ramb18e1.pb_type.xml" xpointer="xpointer(pb_type/metadata/meta[not(@name='fasm_params')])" />
+       <meta name="fasm_params">
+        <xi:include href="ramb18e1.pb_type.xml" xpointer="xpointer(pb_type/metadata/meta[attribute::name='fasm_params']/child::node())" />
```

Comment:
Please add a comment here and in ramb18e1.pb_type.xml about why this is being done